Dockerfile

```diff
@@ -1,5 +1,24 @@
-from ubuntu
-add ./Kinsing.so /Kinsing.so
-add ./run.sh /run.sh
-run chmod 777 /run.sh
-entrypoint /run.sh
+FROM node:20-slim
+
```

Comment:
This line previously contained the malicious `Kinsing.so` file that was being injected into the container. The new Dockerfile completely removes this malware component and replaces it with a proper Node.js application.